packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -47,7 +46,14 @@ export default class QuanticResultLink extends LightningElement {
    * @defaultValue `'search'`
    */
   @api useCase = 'search';
-
+  /**
+   * A list of fields to include in the query results, separated by commas.
+   * @api
+   * @type {string}
+   * @defaultValue `'date,author,source,language,filetype,parents,sfknowledgearticleid'`
```

Comment:
need to update the documented default value as well. They should always be in sync.

```suggestion
   * @defaultValue `'date,author,source,language,filetype,parents,sfknowledgearticleid,sfid,sfkbid,sfkavid'`
```